packages/create-sitecore-jss/src/templates/nextjs-personalize/src/components/CdpIntegrationScript.tsx

```diff
@@ -5,10 +5,11 @@ import { useEffect } from 'react';
 
 declare const _boxeverq: any;
 declare const Boxever: any;
+declare const currentHostname : any;
```

Comment:
`any` type should be removed and replaced by the appropriate type

Reply:
What is the purpose of using `currentHostname`?  It will be used in case if POS is not defined?

Reply:
It should work (as long as the hostname is listed on Boxever of course), this approach is referenced in Boxever doc:
https://doc.sitecore.com/cdp/en/developers/sitecore-customer-data-platform--data-model-2-1/javascript-tagging-examples-for-webpages.html
currentHostname is populated in the script we load via the control